webpack/assets/javascripts/react_app/components/FactCharts/FactChartConstants.js

```diff
@@ -1,3 +1,4 @@
+export const FACT_CHART = 'FACT_CHART';
```

Comment:
I wonder if we should also create an helper to generate automatic ACTION_TYPES:
`REQUEST, SUCCESS, FAILURE` and than remove all of this consts in each component.